ocs_ci/utility/ssl_certs.py

```diff
@@ -266,6 +270,7 @@ def configure_custom_ingress_cert():
         '--type=merge -p \'{"spec":{"defaultCertificate": {"name": "ocs-cert"}}}\''
     )
     exec_cmd(cmd)
+    wait_for_machineconfigpool_status("all", timeout=1800)
```

Comment:
we need to sleep here for 10 sec and then we can start for wait_for_machineconfigpool_status

Reply:
@prsurve why is the sleep necessary? / If it's always necessary should the `wait_for_machineconfigpool_status` function handle that wait time for us?

Reply:
It's because mcp takes few sec to update with new config

Reply:
I added it directly to the function in next commit and removed from all other places as I noticed we are waiting 60 seconds before calling this function on all the places.